workspaces/homepage/plugins/dynamic-home-page/src/components/TemplateSection/TemplateSection.tsx

```diff
@@ -0,0 +1,185 @@
+        />
+      </WarningPanel>
+    );
+  } else {
+    content = (
+      <Box sx={{ padding: '20px 10px 10px 0' }}>
+        <React.Fragment>
+          <Grid container spacing={1} alignItems="stretch">
+            {templates?.items.map((item: any) => (
+              <Grid item xs={12} md={3} key={item.title}>
+                <TemplateCard
+                  link={`/create/templates/default/${item.metadata.name}`}
+                  title={item.metadata.title}
+                  description={item.metadata.description}
+                  kind="Template"
+                />
+              </Grid>
+            ))}
+            {templates?.items.length === 0 && (
+              <Grid item xs={12} md={12}>
+                <Box
+                  sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    border: muiTheme =>
+                      `1px solid ${muiTheme.palette.grey[400]}`,
+                    borderRadius: 3,
+                    overflow: 'hidden',
+                  }}
+                >
+                  <CardContent
+                    sx={{
+                      margin: '1rem',
+                    }}
+                  >
+                    <div className={classes.title}>No templates added yet</div>
+                    <div className={classes.description}>
+                      Once templates are added, this space will showcase
+                      relevant content tailored to your experience.
+                    </div>
+                    <Box>
+                      <Link
+                        to="/catalog-import"
+                        underline="none"
+                        className={classes.link}
+                      >
+                        <div
+                          style={{
+                            display: 'flex',
+                            justifyContent: 'center',
+                          }}
+                        >
+                          Register a template
+                        </div>
+                      </Link>
+                    </Box>
+                  </CardContent>
+                </Box>
+              </Grid>
+            )}
+          </Grid>
+          <Box sx={{ pt: 2 }}>
+            {templates?.items.length > 0 && (
+              <ViewMoreLink to={catalogTemplatesLink} underline="always">
+                View all {templates?.totalItems ? templates?.totalItems : ''}{' '}
+                templates
+              </ViewMoreLink>
+            )}
+          </Box>
+        </React.Fragment>
+      </Box>
+    );
+  }
+
+  return (
+    <Box
+      component={Paper}
+      sx={{
+        padding: '24px 24px 0 24px',
+        border: muiTheme => `1px solid ${muiTheme.palette.grey[300]}`,
+        borderRadius: 3,
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          fontSize: '1.5rem',
```

Comment:
Can we use a h2 or a Typography tag here?

This is visually a headline, then we should also use a markup that makes it more a11y.